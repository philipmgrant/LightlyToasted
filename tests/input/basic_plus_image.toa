Testing with
an example image
|ImageSource: file:///z:/foo/bar.jpg