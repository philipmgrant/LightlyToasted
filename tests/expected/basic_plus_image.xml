<toast><visual><binding template="ToastImageAndText02"><image id="1" src="file:///z:/foo/bar.jpg"/><text id="1">Testing with</text><text id="2">an example image</text></binding></visual></toast>